### ISSUE TITLE
Ist there a way to filter the YouTube search box with keywords? It would be great if that can be possible

### ISSUE THREAD

**Issue by cekdo744** [ulyngs/open-source-no-distractions-for-youtube#63]

**Comment by ulyngs** [comment]
I think this is a great idea --- however, I expect that it would take quite a bit of engineering to pull off, if it's even possible.

It should be simple enough to keep track of a list of keywords that the user adds that they want to filter out, but I expect that the filtering itself will be very challenging to implement: I'm not sure my extension can to access the actual contents of the search box, because it's dynamically generated. If that can be solved, then I'm also not entirely sure how we would want to dynamically hide just those results that contain the keywords. 

But I totally agree with you that this would be great feature to have, so I will think about it! I'm super busy in the next four months or so, but I'll try to find a gap.

Cheers,
Ulrik

**Comment by cekdo744** [comment]
Yes, that would be great! In either case, I am really grateful for what you have brought for us Mac users in this distracting world. Thank you very much for your efforts!
